cmd/juju/model/configcommand.go

```diff
@@ -301,43 +301,63 @@ func (c *configCommand) setConfig(client configCommandAPI, ctx *cmd.Context) err
 // get writes the value of a single key or the full output for the model to the cmd.Context.
 func (c *configCommand) getConfig(client configCommandAPI, ctx *cmd.Context) error {
 	if len(c.keys) == 1 && certBytes != nil {
-		ctx.Stdout.Write(certBytes)
+		_, _ = ctx.Stdout.Write(certBytes)
 		return nil
 	}
-	attrs, err := client.ModelGetWithMetadata()
+	attrs, err := c.getSkimmedModel(client)
 	if err != nil {
 		return err
 	}
+	attrs, err, finished := c.handleIsKeyOfModel(attrs, ctx)
+	if err != nil {
+		return err
+	} else if attrs != nil && finished {
+		return c.out.Write(ctx, attrs)
+	} else if len(c.keys) > 0 && !finished {
+		if isFileLike(c.keys[0]) {
+			return errors.Errorf("value: %q seems to be a file but not found", c.keys[0])
```

Comment:
can we remove the "value: " part of the message please.

Reply:
:+1: